### PR TITLE
Removes an assembly-brained note about multiplication from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
image

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -470,8 +470,6 @@ The `SHOULD_CALL_PARENT(TRUE)` lint should be added to ensure that overrides/chi
 
 * You are expected to help maintain the code that you add, meaning that if there is a problem then you are likely to be approached in order to fix any issues, runtimes, or bugs.
```
Comment (MrDoomBringer) [inline .github/CONTRIBUTING.md:472]:
```suggestion

* Do not divide when you can easily convert it to multiplication. (ie `4/2` should be done as `4*0.5`)

```
this should help clarify your PR a bit

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Fikou) [inline .github/CONTRIBUTING.md:472]:
```suggestion
* Do not divide when you can easily convert it to multiplication. (ie `4/2` should be done as `4*0.5`)

```

**Comment by ExcessiveUseOfCobblestone** [comment]
Pov u r banned from dividing

**Comment by interestingusernam3** [comment]
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US
HE WILL NOT DIVIDE US

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (MrDoomBringer) [reply .github/CONTRIBUTING.md:472]: Bad suggestion, this needlessly fgets rid of the empty line 472

**Comment by coiax** [comment]
If I can't tell whether the PR is serious or a meme, then you probably need to try harder.

**Pull request closed without merging**